custom/opm/beneficiary.py

```diff
@@ -759,33 +765,42 @@ class ConditionsMet(OPMCaseRow):
         ('cash', _("Payment Amount"), True),
         ('case_id', _('Case ID'), True),
         ('closed_date', _("Closed On"), True),
-        ('payment_last_month', _("Payment amount received last month"), True),
+        ('payment_last_month', _("Payment amount received last month (Rs.)"), True),
         ('cash_received_last_month', _("Cash received last month"), True),
     ]
 
     def __init__(self, case, report, child_index=1, **kwargs):
         super(ConditionsMet, self).__init__(case, report, child_index=child_index, **kwargs)
         self.serial_number = child_index
-        self.payment_last_month = self.last_month_row.total_cash if self.last_month_row else 0
+        self.payment_last_month = "Rs.%d" % self.last_month_row.total_cash if self.last_month_row else 0
         self.cash_received_last_month = self.last_month_row.vhnd_available_display if self.last_month_row else 'no'
+        self.awc_code = CommCareUser.get_by_user_id(case.user_id).user_data.get('awc_code', EMPTY_FIELD)
```

Comment:
This will require lot of individual couch calls. It would be ideal to get this from the result of Fluff Calculations. You can potentially use [this](https://github.com/dimagi/commcare-hq/pull/5834/files#diff-a7e5dd5cadeed32505efbf20c57da3e4R94)